sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -149,6 +150,17 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
     }
   }, [])
 
+  const scrollToVerify = () => {
+    document.getElementById("isVerifiedContainer").scrollIntoView({ behavior: "smooth" })
+  }
+
+  useEffect(() => {
+    if (jumpToVerify && tabIndex === 0) {
+      scrollToVerify()
+      setJumpToVerify(false)
+    }
+  }, [tabIndex])
```

Comment:
I think at the very least we should add jumpToVerify to this, we should probably add the other dependencies to this as well I think

Reply:
Can you walk me through the why if we only want this use effect to run when tabIndex changes?

Reply:
@emilyjablonski , `jumpToVerify` is the only other needed dependency here, since we're checking for that and tabIndex`.